scripts/fbi/hate_crime/geo_id_resolver.py

```diff
@@ -0,0 +1,270 @@
+    'amherst town ny': '3602902000',
+    'ventura ca': '0665042',
+    'ramapo town ny': '3608760510',
+    'canton township mi': '2613120',
+    'clarkstown town ny': '3608715968',
+    'west bloomfield township mi': '2612585480',
+    'weymouth ma': '2578972',
+    'hempstead village ny': '3633139',
+    'irvington nj': '3401334450',
+    'bloomfield nj': '3401306260',
+    'west orange nj': '3401379800',
+    'redford township mi': '2616367625',
+    'greenburgh town ny': '3611930367',
+    'barnstable ma': '2503690',
+    'freeport village ny': '3627485',
+    'de kalb il': '1719161',
+    'meridian township mi': '2606553140',
+    'greenacres city fl': '1227322',
+    'saginaw township mi': '2614570540',
+    'pittsfield township mi': '2616164560',
+    'montclair nj': '3401347500',
+    'commerce township mi': '2612517640',
+    'orangetown town ny': '3608755211',
+    'haverstraw town ny': '3608732765',
+    'yorktown town ny': '3611984077',
+    'blackman township mi': '2607508760',
+    'independence township mi': '2612540400',
+    'belleville nj': '3401304695',
+    'orion township mi': '2612561100',
+    'bethlehem town ny': '3600106354',
+    'carmel town ny': '3607912529',
+    'guilderland town ny': '3600131104',
+    'houma la': '2236255',
+    'franklin ma': '2525172',
+    'spring valley village ny': '3670420',
+    'mount lebanon pa': '4200351696',
+    'paso robles ca': '0622300',
+    'white lake township mi': '2612586860',
+    'orange city nj': '3401313045',
+    'orchard park town ny': '3602955277',
+    'port chester village ny': '3659223',
+    'la canada flintridge ca': '0639003',
+    'espanola nm': '3525170',
+    'kensington ca': '0638086',
+    'sansom park village tx': '4865660',
+    'broadmoor ca': '0608338',
+    'carmel ca': '0611250',
+    'angels camp ca': '0602112',
+    'port mansfield tx': '4858928'
+}
+
+with open(os.path.join(_CITY_GEOCODES_PATH), encoding='utf8') as csvfile:
+    csv_reader = csv.reader(csvfile)
+    for a in csv_reader:
+        _CITY[a[0]] = a[1]
+
+
+def _state_to_dcid(state_code: str) -> str:
+    """
+    Takes in the state alpha2 code and returns the dcid.
+
+    Args:
+        state_code: The alpha2 code of the state.
+
+    Returns:
+        The dcid of the state in DC. Returns an empty string if the state is not
+        found.
+    """
+    if state_code in _IGNORE_STATE_ABBR:
+        return ''
+
+    if state_code in _US_GEO_CODE_UPDATE_MAP:
+        state_code = _US_GEO_CODE_UPDATE_MAP[state_code]
+
+    if state_code in USSTATE_MAP:
+        return USSTATE_MAP[state_code]
+    else:
+        return ''
+
+
+def _get_county_variants(county: str) -> list:
+    """
+    Given a county, returns a list of the possible variants of the county name.
+
+    Args:
+        county: A string literal that represents the county name.
+
+    Returns:
+        A list of the possible variants of the county name.
+    """
+    county = county.replace(' County Police Department', ' County')
```

Comment:
Since ' County' suffix is added below, should the suffix `'County Police Department'` be removed (not replaced)

Reply:
Done. Thanks for pointing that out.